app/assets/stylesheets/application.scss

```diff
@@ -2,6 +2,7 @@ $govuk-global-styles: true;
 $govuk-new-link-styles: true;
 $govuk-new-typography-scale: true;
 $govuk-assets-path: "/";
+$govuk-suppressed-warnings: "legacy-organisation-colours";
```

Comment:
Are we using any legacy colours?
If yes, can we update them?

Reply:
We are not, but I have changed this to adopt the new colours